packages/sitecore-jss-angular/tsconfig.spec.json

```diff
@@ -0,0 +1,17 @@
+{
+    "extends": "./tsconfig.json",
+    "compilerOptions": {
+      "outDir": "./dist/out-tsc",
+      "types": [
+        "jasmine"
+      ]
+    },
+    "files": [
+      "./src/test.ts"
+    ],
+    "include": [
+      "**/*.spec.ts",
+      "**/*.d.ts"
+    ]
+  }
+  
```

Comment:
Add new line

Reply:
there's the line at the end - but I fixes the curly bracket's position